x-pack/plugins/observability_solution/apm/server/routes/transactions/__snapshots__/queries.test.ts.snap

```diff
@@ -0,0 +1,406 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Do we need this file? 🤔 Isn't it a duplicate of [x-pack/solutions/observability/plugins/apm/server/routes/transactions/__snapshots__/queries.test.ts.snap](https://github.com/elastic/kibana/pull/216679/files#diff-1286ab4b783ade0daeca30f2599c8c2bb9d6518c926aa84ee258a207792b7670)

Reply:
ah I think so....